.github/workflows/test.yml

```diff
@@ -88,7 +86,11 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -U pip setuptools wheel
+          sed -i -E "s/^Django==.+$/Django==${{ matrix.django-version }}.*/" ./requirements.txt
```

Comment:
Not sure if there's a nicer way to override Django version from requirements.txt?

Reply:
There is: https://github.com/wemake-services/django-test-migrations/blob/1f002997b50619d39e10b32693a8f8727c04f175/.github/workflows/test.yml#L110-L112

Reply:
I don't understand how I would use that here. That example is not using a `requirements.txt`.

Reply:
It is using `pyproject.toml`, which is the same thing. Basically, you install `requirements.txt` as usual, then you run `pip install -U $DJANGO_VERSION_OVERRIDE` to change the django version for this build.

Reply:
Ah OK, the `poetry install` line wasn't highlighted in your link.

It bothers me a little that Django would then be installed twice. And I suspect when using multiple `pip` invocations, it won't error on dependency conflicts.

Do you think it's an improvement?

Reply:
> when using multiple pip invocations, it won't error on dependency conflicts.

Actually this can be solved by using `pip check`, which returns nonzero on conflicts.

Reply:
Yep, I think it looks better this way. Installing Django twice is ugly but the better of two evils :)